.github/workflows/search.config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
We might want to put this file in a different location since it's not really a workflow

Reply:
I've moved it to a new config dir.